src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
Similarly, the message 'Truck stopped working' should be more informative about the type of work the Truck was doing, to comply with the checklist guidelines.